services/src/main/scala/cromwell/services/womtool/impl/WomtoolServiceInCromwellActor.scala

```diff
@@ -0,0 +1,81 @@
+package cromwell.services.womtool.impl
+
+import akka.actor.{ActorRef, Props}
+import cats.data.Validated.{Invalid, Valid}
+import com.typesafe.config.Config
+import cromwell.core.Dispatcher.ServiceDispatcher
+import cromwell.core.WorkflowSourceFilesCollection
+import cromwell.languages.util.ImportResolver.HttpResolver
+import cromwell.languages.LanguageFactory
+import cromwell.languages.util.{ImportResolver, LanguageFactoryUtil}
+import cromwell.services.womtool.WomtoolServiceActor
+import cromwell.services.womtool.WomtoolServiceMessages._
+import cromwell.util.GracefulShutdownHelper.ShutdownCommand
+import wom.core.WorkflowSource
+import wom.expression.NoIoFunctionSet
+
+class WomtoolServiceInCromwellActor(serviceConfig: Config, globalConfig: Config, serviceRegistryActor: ActorRef) extends WomtoolServiceActor {
+
+  override def receive: Receive = {
+    case DescribeRequest(filesCollection) =>
+      sender ! describeWorkflow(filesCollection)
+    case ShutdownCommand =>
+      // This service doesn't do any work on shutdown but the service registry pattern requires it (see #2575)
+      context.stop(self)
+  }
+
+  def describeWorkflow(wsfc: WorkflowSourceFilesCollection): DescribeResult = {
+
+    // The HTTP resolver is used to pull down workflows submitted by URL
```

Comment:
Would it make sense to always have `HttpResolver` built into the `findWorkflowSource` method?